cmd/build/v2/config.go

```diff
@@ -84,3 +86,13 @@ func (oc oktetoBuilderConfig) GetGitCommit() string {
 func (oc oktetoBuilderConfig) GetAnonymizedRepo() string {
 	return oc.repository.GetAnonymizedRepo()
 }
+
+func (oc oktetoBuilderConfig) GetBuildContextHash(buildInfo *model.BuildInfo) string {
+	buildContext := buildInfo.Context
+	treeHash, err := oc.repository.GetTreeHash(buildInfo.Context)
+	if err != nil {
+		oktetoLog.Info("error trying to get tree hash for build context '%s': %w", buildContext, err)
```

Comment:
What do you think if we return `err` and log outside the function? I am concerned it will be harder to test and considering the effort in refactoring the logger to be injected instead of being global, it could be one less instance in the code to change in the future.

Reply:
tbh Im not sure where to move it. I've followed the same approach we have now for `configRepositoryInterface` interface like `GetGitCommit` where we just return a string with the value, I think we should try to be as consistent as possible